met-web/src/services/widgetService/MapService/index.tsx

```diff
@@ -41,3 +43,23 @@ export const postMap = async (widget_id: number, data: PostMapRequest): Promise<
         return Promise.reject(err);
     }
 };
+
+interface PreviewShapefileRequest {
+    file?: File | undefined;
```

Comment:
when you add '?' it treats it as whatever | undefined already. So we can safely remove | undefined here